### ISSUE TITLE
23.05 Retros Action Items: Are we going to include Features Grid for the next release?

### ISSUE THREAD

**Issue by vincent6767** [o3de/tsc#85]
Based on [this ](https://docs.google.com/spreadsheets/d/1jvPxKBBQTSfkS3jg570jZ9snFaQUUyAtUQu9SBwC1Xc/edit?pli=1#gid=0&range=A9)retros sheet, @nick-l-o3de will hold a discussion with the TSC about this topic. Potentially add in General Guidelines.

**Comment by vincent6767** [comment]
@nick-l-o3de Are there any updates around this? 

CC: @tonybalandiuk

**Comment by vincent6767** [comment]
Any updates on this? @nick-l-o3de

**Comment by nick-l-o3de** [comment]
I'm going to have to go back and listen to the recoding but my understanding is that yes, we want to get the feature grid available, but we are not going to block releases and make it an official part of the release **until** Royal either fixes the format so that its just copy pasteable into the docs as-is, or deputizes someone to do that on his behalf.

Docs/Community is to reach out to Royal to make sure he has an example of the exact format required so that he can update his generator to produce exactly that, so the process is just one click -> copy into docs.   Then once it is verified to be working well, it should be straightforward to add an item to the release checklist that is just to go there and capture it for the release (probably as part of when it branches, so that it captures a snapshot of the features at time of branch?) and how to actually physically execute the copy and paste (what to click, where to put things.)

Sound ok?

**Comment by tonybalandiuk** [comment]
makes sense @nick-l-o3de . @chanmosq  can you reach out to @OBWANDO about the format gap(s) for the feature grid export?

**Comment by chanmosq** [comment]
@tonybalandiuk Yes, I'll speak with Royal about it

**Comment by chanmosq** [comment]
I've written the pain points for updating the feature grid in this document: [Improving the feature grid process](https://docs.google.com/document/d/1Lz7MbopIl3vABvZJ0YiRK9xltWzoDNiX8yKU_oD48Tk/edit?usp=sharing)

**Comment by vincent6767** [comment]
I'll close this issue since we have achieved the main goals of this issue as stated here: https://github.com/o3de/tsc/issues/85#issuecomment-1638318652

Thanks, everyone!

@chanmosq Please create another GHI on the SIG docs community to track the follow-up with Royal.

**Comment by chanmosq** [comment]
@vincent6767 I've pass on my notes about feature grid pain points to Naomi and Royal. Would you like me to share the notes somewhere in GitHub as well?

**Comment by vincent6767** [comment]
My understanding is SIG docs community will work with Royal to improve the features grid tools improvement and the Definition of Done is the tools are updated based on the discussion. Hence I suggest SIG Docs community track this as GHI in their board. @chanmosq

**Comment by chanmosq** [comment]
@vincent6767 SIG docs community doesn't have the resources to improve the feature grid tool. So I don't think the proposed ticket will get any action in the SIG docs community board. Instead I passed the information to O3DF, identifying it as a gap in the docs workflows. 

To clarify, there is a process to use and update the feature grid for the upcoming release. So this isn't a blocker. It's just that from our experience we identified ways to improve it.
